web/yaamp/modules/site/common_results.php

```diff
@@ -176,6 +186,55 @@ function cmp($a, $b)
 	echo '<td align="right" style="font-size: .8em; '.$style.'">'.$btcmhday1.'</td>';
 
 	echo '</tr>';
+	
+	if ($algo_selected == $algo && $stratum_instances > 0)	{
+		echo '<tr>';
+		echo '<td colspan="13">';
+		showTableSorter('stratumstable', '{
+			tableClass: "dataGrid",
+			widgets: ["Storage","saveSort"],
+			textExtraction: {
+				1: function(node, table, cellIndex) { return $(node).attr("data"); }
+			},
+			widgetOptions: {
+				saveSort: true
+			}}');
+		echo <<<END
+		<table>
```

Comment:
showTableSorter() php func creates a `<table class="dataGrid">`... this one is double/useless so